### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by xanecs**
Add a Dockerfile that builds the react app and runs it in an nginx container.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:10 AS builder
+RUN mkdir /app
+ADD . /app
```
Comment (TheKangaroo) [inline Dockerfile]: we should use COPY for that

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+WORKDIR /app
+RUN npm install
+RUN . /app/.env.production && npm run build
+
+FROM nginx:alpine AS server
```
Comment (TheKangaroo) [inline Dockerfile]: We could use the unpivileged, non-root image: https://hub.docker.com/r/nginxinc/nginx-unprivileged

File: README.md [hunk 3]
```diff
@@ -70,6 +70,24 @@ $ npm test
...
+```
+$ docker build -t my-lisa-image .
+```
+This multi-stage docker build will install all dependencies,
+build the production bundle and copy it to a ngninx container.
+
+Then you can run it.
+
+```
+$ docker run -p 8080:80 my-lisa-image
```
Comment (TheKangaroo) [inline README.md]: With non-root:
$ docker run -p 8080:8080 my-lisa-image

File: Dockerfile [hunk 4]
```diff
@@ -1,9 +1,9 @@
 FROM node:10 AS builder
 RUN mkdir /app
-ADD . /app
+COPY source dest . /app
```
Comment (laugmanuel) [inline Dockerfile]: Shouldn't this one be `COPY . /app`?

Comment (xanecs) [reply Dockerfile]: Yes...

**Comment by timogoebel** [comment]
Thanks. Can we add a CI test to verify this actually works?

**Comment by timogoebel** [comment]
Thanks, @xanecs.

**Pull request merged**